spec/services/hashtag_autocomplete_service_spec.rb

```diff
@@ -307,6 +307,14 @@
     end
   end
 
+  describe "#find_by_ids" do
+    it "can return only category" do
```

Comment:
```suggestion
    it "looks up and returns the right category" do
```

Would this better describe what the test is asserting for because I wasn't quite sure based on the current description.